templates/web/base/maps/wms_config.html

```diff
@@ -0,0 +1,17 @@
+<script nonce="[% csp_nonce %]">
+  var fixmystreet = window.fixmystreet || {};
+  fixmystreet.wms_config = {
+        'map_projection': '[% map.map_projection %]',
+        'tile_dpi': [% map.tile_dpi %],
+        'tile_urls': [% map.tile_urls | safe %],
+        'tile_size': [% map.tile_size %],
+        'tile_suffix': '[% map.tile_suffix %]',
```

Comment:
There is something wrong here, as this is never set by the server side. Is this line needed (and therefore formatSuffix)?